src/averageAges.js

```diff
@@ -20,6 +20,15 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+
+  const men = people.filter(person => person.sex === 'm'
+  && (century === Math.ceil(person.died / 100) || !century));
+
+  const average = men
```

Comment:
```suggestion
  const averageAge = men
```
Let's specify to make the variables more descriptive. Please fix all cases